app/views/users/_institutions.html.erb

```diff
@@ -1,17 +1,21 @@
 <script>
-$(document).ready(function(){
-  $('#user_institution_id').change(function() {
-    if(this.value == "") {
-      $('#institution_name').show();
-      $('#name_of_other_institution').show();
-    }else{
-      $('#institution_name').hide();
-      $('#name_of_other_institution').hide();
-    }
-  });
-});
+    $(document).ready(function(){
```

Comment:
Were these white space changes (lines 2-12) made because CodeClimate was complaining? If not, then I don't think these are necessary.

Reply:
I have no idea where these lines came from and I do not remember putting them here.